rero_ils/modules/patrons/api.py

```diff
@@ -586,7 +589,8 @@ def can_request(cls, item, **kwargs):
         if patron.is_blocked:
             messages.append(patron.get_blocked_message())
         if patron.is_expired:
-            messages.append(_('Patron rights expired.'))
+            messages.append(_('Your account has expired. '
+                              'Please contact your library.'))
```

Comment:
seems wrong because this method could be used also by API for the Pro UI interface.